packages/components/src/progress_bar/index.jsx

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { getPercentageString } from './utils';
+import styles from './index.module.scss';
+
+export const THEME_MODERN = 'modern';
```

Comment:
1. missing constant for THEME_LEGACY
2. watch out: named constants will be exported 1:1 to the final esm bundle. Better name it something like `PROGRESS_BAR_THEME_MODERN` - check other named exports in components package for more examples